aws_lambda_builders/utils.py

```diff
@@ -196,3 +196,28 @@ def create_symlink_or_copy(source: str, destination: str) -> None:
             exc_info=ex if LOG.isEnabledFor(logging.DEBUG) else None,
         )
         copytree(source, destination)
+
+
+def _is_within_directory(directory, target):
```

Comment:
nit: can we add type hinting for the params and return value for these new methods?

Reply:
sure, can do